utils/nxdl2rst.py

```diff
@@ -792,13 +786,13 @@ def main():
         print(f"Cannot find {nxdl_file}")
         exit()
 
-    print_rst_from_nxdl(nxdl_file)
+    subdir = os.path.basename(os.getcwd())
+    print_rst_from_nxdl(nxdl_file, subdir)
```

Comment:
In https://github.com/nexusformat/definitions/pull/1151 (a MR that address the same issue) I derive the subdir from `nxdl_file` inside the `print_rst_from_nxdl` function.